metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -329,12 +330,17 @@ def get_log_stream(job):
                         else:
                             pass
                     else:
+                        i = 0
                         yield line
+                return
             except Exception as ex:
+                exception = ex
                 if self.is_crashed:
                     break
-                sys.stderr.write(repr(ex))
-                time.sleep(2 ** i)
+                #sys.stderr.write(repr(ex) + '\n')
```

Comment:
This is intentionally left out in the code so that we can easily debug issues while in dev mode. The raised exception in L:343 will convey the last generated exception to the user.